experimental/language-generation/csharp_dotnetcore/06.using-cards/AdapterWithErrorHandler.cs

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT License.
 
 using System;
+using System.IO;
 using Microsoft.Bot.Builder;
-using Microsoft.Bot.Builder.Integration.AspNet.Core;
-using Microsoft.Bot.Connector.Authentication;
 using Microsoft.Extensions.Logging;
+using Microsoft.Bot.Connector.Authentication;
 using Microsoft.Bot.Builder.LanguageGeneration;
-using System.IO;
-using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using ActivityBuilder = Microsoft.Bot.Builder.Dialogs.Adaptive.Generators.ActivityGenerator;
```

Comment:
ActivityBuilder  replaces MessageFactory? for the future, when we GA Adaptive, are the two going to work together?

Reply:
That's correct. When we GA, the GenerateFromLG will be part of `MessageFactory`